plugins/generator-1.17.1/forge-1.17.1/templates/plant/plant.java.ftl

```diff
@@ -41,7 +41,7 @@ import net.minecraft.sounds.SoundEvent;
 
 public class ${name}Block extends <#if data.plantType == "normal">Flower<#elseif data.plantType == "growapable">SugarCane<#elseif data.plantType == "double">DoublePlant</#if>Block<#if data.hasTileEntity> implements EntityBlock</#if>{
 	public ${name}Block() {
-		super(<#if data.plantType == "normal">${generator.map(data.suspiciousStewEffect, "effects")}, ${data.suspiciousStewDuration},</#if>
+		super(<#if data.plantType == "normal">${data.suspiciousStewEffect?starts_with("CUSTOM:")?then("MobEffects.SATURATION", generator.map(data.suspiciousStewEffect, "effects"))}, ${data.suspiciousStewDuration},</#if>

```

Comment:
We should be able to set effect here directly for custom effects too, same for 1.16.5

Reply:
This works in 1.17, but in 1.16 this crashed Minecraft while loading

Reply:
I see, it was probably due to the registry loading order